docs/api.md

```diff
@@ -71,54 +71,48 @@ _Will be removed in v3.0.0._
 
 ## Configuration
 
-### #component_parent_class
-
-Parent class for generated components
-
-    config.view_component.component_parent_class = "MyBaseComponent"
-
-Defaults to "ApplicationComponent" if defined, "ViewComponent::Base" otherwise.
-
 ### #default_preview_layout
 
 Set a custom default layout used for preview index and individual previews:
 
     config.view_component.default_preview_layout = "component_preview"
 
-### #generate_distinct_locale_files
+### #generate
 
-Always generate as many translations files as available locales:
+Configuration for generators
 
-    config.view_component.generate_distinct_locale_files = true
+Defaults to `false` unless otherwise stated.
```

Comment:
This doesn't seem right. Isn't this just a namespace, not an option?

Reply:
I see where you're coming from - this perhaps isn't phrased accurately to what I was intending. What I meant by this was that all options under the namespace default to `false` unless otherwise stated.

```suggestion
All options under this namespace default to `false` unless otherwise stated.
```